package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "okx-api",
-  "version": "1.5.1",
+  "version": "1.5.11",
```

```suggestion
  "version": "1.5.2",
```

Only increment by one digit please.